src/www/app/app.ts

```diff
@@ -550,10 +551,19 @@ export class App {
   //#region UI dialogs
 
   private showConfirmationDialog(message: string): Promise<boolean> {
-    // Temporarily use window.alert and window.confirm here
+    // Temporarily use window.confirm here
     return new Promise<boolean>(resolve => resolve(confirm(message)));
   }
 
+  private showErrorDetailDialog(error: Error) {
+    // Temporarily use window.alert here
+    if (error.stack) {
```

Comment:
I'm not sure the stack is helping much. See above for my suggestion to show name, message and cause.